containers/images/hawkular-mock-agent-inventory/hawkular-mock-agent-inventory.py

```diff
@@ -30,26 +31,52 @@
 
 class AgentLifeCycle(TaskSet):
     def on_start(self):
+        global feed
         self.feed_id = "feed-" + str(uuid.uuid4())
-        print "New Agent [%s]" %  self.feed_id,
+        feed = self.feed_id
+        print "New Agent [%s]" % self.feed_id,
```

Comment:
I don't know if the agent can be configured to use a log system and perhaps this print I put here for debug can be enable/disable per configuration.
Anyway, no need to re-arrange anything for this on this PR, just a note.

Reply:
Same for all print calls.

Reply:
There is an option called ---logfile which allow Path to log file. If not set, log will go to
stdout/stderr. I am ok with theses prints because on the openshift deployment, it can very useful to diagnosis the pod state